dbt/tracking.py

```diff
@@ -241,11 +239,11 @@ def track_invalid_invocation(
 
     user = active_user
     invocation_context = get_invocation_invalid_context(
-            user,
-            project,
-            args,
-            result_type,
-            result
+        user,
+        project,
+        args,
+        result_type,
+        result
```

Comment:
I think we probably also want to nix `result` here too, right? This would contain the exception that dbt throws if the project is misconfigured. There are a couple of instances where we pass exceptions in around [here](https://github.com/fishtown-analytics/dbt/blob/development/dbt/main.py#L200)